lessons/14_async/index.md

```diff
@@ -112,4 +112,32 @@ Async function with error handling
 })();
 ```
 
-## Fetch!
+## Fetch
+
+Resource Links:
+
+- [Docs](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)
+- [Tutorial](https://javascript.info/fetch)
+
+## Classwork
+
+Lets go directly to playing with fetch(). Often times when working with APIs we need to call serval in order to preform some sort of operation like logging in. Today we are going to do something similar with a multi step math workflow. You are free to use promises, async functions, and callbacks as you with.
+
+1. Start by putting some status text on the page ex: **loading...**
+2. Get three random numbers from `/random` endpoint with a GET request
```

Comment:
where those endpoints are pointing?

Reply:
added full documentation of classwork including full urls and instructions